### PR TITLE
specify that include/exclude use the directory names

### PR DESCRIPTION
**Pull request by alwaysintreble**
i tried using the worlds' game name string :)

### CODE REVIEW

File: ap-roller.py [hunk 1]
```diff
@@ -258,7 +259,7 @@ def main(args: "argparse.Namespace"):
     py_args = '-O' if args.optimize else []
     for ap in aps:
         assert os.path.isdir(str(ap)), f"AP directory or venv does not exist: '{ap}'"
-    output_dir = os.path.join("output", str(os.getpid()))
+    output_dir = os.path.join("output", f"{os.getpid()}_{datetime.now().strftime('%m_%d-%H_%M-%S')}")
```
Comment (black-sliver) [inline ap-roller.py]: I like how you smuggled the dir name change in there :P
Maybe move pid to the back so it's sorted?

File: ap-roller.py [hunk 2]
```diff
@@ -145,9 +146,9 @@ def collect_yamls(mode, max_slots, include=None, exclude=None, limit=1000) -> Li
     :param max_slots:
       maximum number of slots in each set
     :param include:
-      list of games to include, None if all games should be included
+      list of games to include, None if all games should be included, using directory names
```
Comment (black-sliver) [inline ap-roller.py:149]: should probably also change it in the argparse section?

File: ap-roller.py (hunk 1 above)
Comment (alwaysintreble) [reply ap-roller.py]: yeah i realized how annoying it was to check against generations so added this in and just pushed it to the same branch and forgot to change the title lol. will do

**Comment by black-sliver** [comment]
Thanks :-)

**Pull request merged**